Makefile

```diff
@@ -499,7 +499,7 @@ push-bundle-to-quay:
 .PHONY: dev-release
 ## validating the operator by installing new quay releases
 dev-release:
-	BUNDLE_VERSION=$(BUNDLE_VERSION) ./hack/dev-release.sh
+	INDEX_IMAGE="quay.io/$(OPERATOR_GROUP)/$(GO_PACKAGE_REPO_NAME)-index:$(BUNDLE_VERSION)" BUNDLE_VERSION=$(BUNDLE_VERSION) ./hack/dev-release.sh
```

Comment:
Where do we build the index image? Do we really need to have index image per SBO version? I could imaging one index version that gets rebuild with every build to master and its purpose is to keep the last master build.

Reply:
The index image would be build by Travis CI per merge-to-master. Once https://github.com/redhat-developer/service-binding-operator/pull/767 is merged

Reply:
Let's move the discussion about the index image to https://github.com/redhat-developer/service-binding-operator/pull/767